<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageClassificationControlPanel</class>
 <widget class="QWidget" name="ImageClassificationControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="showResults">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show Results</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="imgAmount">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Single Image</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>More Images</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="saveResults">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Results</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseNeuralNetwork">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Neural Networks</string>
   </property>
  </widget>
  <widget class="DeviceCheckBoxManager" name="CheckBoxWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="mode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>High Performance Mode</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>High Efficiency Mode</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Low Power Consumption Mode</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="appendResults">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Append Results</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startImgClass">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Image Classification</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addImage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Image/s</string>
   </property>
  </widget>
  <widget class="QLabel" name="powerPredictionText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Power Prediction:</string>
   </property>
  </widget>
  <widget class="QLabel" name="powerPrediction">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>490</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="performancePredictionText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Performance Prediction:</string>
   </property>
  </widget>
  <widget class="QLabel" name="performancePrediction">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>530</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="deviceSearch">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Search for Devices</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DeviceCheckBoxManager</class>
   <extends>QWidget</extends>
   <header>DeviceCheckBoxManager.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
